$NetBSD: patch-CMakeLists.txt,v 1.9 2026/09/29 16:11:56 bouyer Exp $

export Protobuf_INCLUDE_DIR from protobuf::libprotobuf, for use in
api/CMakeLists.txt

--- CMakeLists.txt.orig	2026-08-28 20:01:37.000000000 +0200
+++ CMakeLists.txt	2026-09-26 11:17:30.991741798 +0200
@@ -906,6 +906,7 @@
     find_package( Protobuf QUIET CONFIG )
     if( Protobuf_FOUND )
         set( Protobuf_LIBRARY "protobuf::libprotobuf" )
+        get_target_property(Protobuf_INCLUDE_DIR protobuf::libprotobuf INTERFACE_INCLUDE_DIRECTORIES)
         set( _protobuf_found TRUE )
         message( STATUS "Found Protobuf using CONFIG mode" )
     else()
